content/partials/firewall.md

```diff
@@ -12,12 +12,13 @@ Codemagic requires direct access to the repositories for the following use cases
 - For cloning the repository to the build machine during the build.
 
 The following IP addresses must be allowed through the firewall (whitelisted):
+
 {{< highlight Shell "style=rrt">}}
+66.185.17.80/29
 199.7.162.128/29
 207.254.42.240/29
 34.74.234.56/32
 35.185.76.207/32
-66.185.17.82/32
```

Comment:
you are saying that 66.185.17.82/32 is the correct address, how come we are removing it from the docs then?

Reply:
I didn't remove it, just moved as the first one

Reply:
ah, my mistake..
Looks good to me now